build-system/compile/compile-wrappers.js

```diff
@@ -56,9 +59,11 @@ exports.extension = function (
     }
     priority = 'p:"high",';
   }
+  // Use a numeric value instead of boolean.
+  isModule = isModule ? 1 : 0;
```

Comment:
Is there a reason to overwrite the previous value versus making a new variable based on the boolean condition?

Reply:
there isn't. ill go ahead and do that for readability 👍